multi_polygon_test.go

```diff
@@ -0,0 +1,19 @@
+package geom
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestMultiPolygon(t *testing.T) {
```

Comment:
Make this table driven.

Reply:
I will add these on a case-by-case basis if we ever reach the point where there are multiple test cases with diverging behaviors. We're not there yet.